spec/Powertrain/FuelSystem.vspec

```diff
@@ -58,18 +57,21 @@
   unit: l/100km
   min: 0
   description: Average consumption in liters per 100 km.
+# The period used to calculate average consumption is OEM-specific
```

Comment:
Do we want to have a better explanation on how the value shall be calculated, or do we want to keep that as OEM-specific. And if so, do we want to explicitly state that it is OEM-specific with a comment like here?

For example - Mercedes (https://developer.mercedes-benz.com/products/connect_your_fleet/details)
has two values: Driving.Consumption.Fuel.SinceStart and Driving.Consumption.Fuel.SinceReset

Volvo has just one value (https://developer.volvocars.com/volvo-api/extended-vehicle/#Average_Fuel_Consumption), not pecifying wther it is for current trip, since last reset or sine the car was produced.